src/commands/fun/poll.ts

```diff
@@ -54,9 +57,12 @@ export default class extends Command {
 		const timespan = parse(interaction.options.getString('timespan'));
 		const question = interaction.options.getString('question');
 		const choices = interaction.options.getString('choices').split('|').map(choice => choice.trim());
+		if (!args.includes(interaction.options.getString('optiontype'))) {
```

Comment:
This is a required field, so why would this be necessary? Just something to satisfy typescript/eslint?

Reply:
This is more for a developer touching it. I do stuff like [this](https://github.com/ud-cis-discord/SageV2/pull/266/files#diff-71480bc3b9d8496b951d2aa274bcda13f86db223ae34c94015a2f5bb3d226f7dR63) a bunch so it's just making sure when I check the option type against `"Single" | "Multiple"` it *actually is* one of those types

Reply:
To be clear, there's probably definitely a better way to constrain stuff later but djs doesn't let us set union types as argument choices yet :(

Reply:
Ok yeah, I see now